thoth-export-server/src/xml/doideposit_crossref.rs

```diff
@@ -465,12 +465,16 @@ impl XmlElementBlock<DoiDepositCrossref> for WorkIssuesSeries {
                 })
             })?;
             write_full_element_block("issn", Some(vec![("media_type", "print")]), w, |w| {
```

Comment:
[ISSNs are required in `series_metadata`](https://data.crossref.org/reports/help/schema_doc/5.3.1/crossref5_3_1_xsd.html#series_metadata) – do not output the `series_metadata` element if there isn't at least one ISSN